<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hestabits-MacBook-Pro.local" failures="5" tests="5" name="safari test" time="71.092" errors="0" timestamp="2024-04-12T13:53:31 IST">
  <testcase classname="UrbanTestCases.TC_002_DDT" name="verify_loginDDT" time="20.507">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at UrbanTestCases.TC_002_DDT.verify_loginDDT(TC_002_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="UrbanTestCases.TC_002_DDT" name="verify_loginDDT" time="9.964">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at UrbanTestCases.TC_002_DDT.verify_loginDDT(TC_002_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="UrbanTestCases.TC_002_DDT" name="verify_loginDDT" time="9.971">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at UrbanTestCases.TC_002_DDT.verify_loginDDT(TC_002_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="UrbanTestCases.TC_002_DDT" name="verify_loginDDT" time="9.976">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at UrbanTestCases.TC_002_DDT.verify_loginDDT(TC_002_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="UrbanTestCases.TC_002_DDT" name="verify_loginDDT" time="9.98">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at UrbanTestCases.TC_002_DDT.verify_loginDDT(TC_002_DDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- safari test -->
